packages/client/hmi-client/src/services/models/simulation-service.ts

```diff
@@ -32,11 +26,59 @@ export async function getRunStatus(runId: number) {
 
 export async function getRunResult(runId: number) {
 	try {
-		const resp = await API.get(`simulation-service/${runId}/result`);
+		const resp = await API.get(`simulation/${runId}/result`);
 		const output = resp.data;
 		return output;
 	} catch (err) {
 		logger.error(err);
 		return null;
 	}
 }
+
+export async function getSimulation(id) {
+	try {
+		const response = await API.get(`/simulation/${id}`);
+		return response.data;
+	} catch (error) {
+		logger.error(error);
+		return null;
+	}
+}
+
+export async function createSimulation(name, description, simulationParams, result) {
+	try {
+		const response = await API.post('/simulation', {
+			name,
+			description,
+			simulationParams,
+			result
+		});
```

Comment:
Would be better to pass a `simulation Simulation` as argument.
```suggestion
export async function createSimulation(simulation: Simulation) {
	try {
		const response = await API.post('/simulation', simulation);
```